File: test_tcc_mobile/app/src/main/java/com/api/test_tcc_mobile/retrofit/UserRetrofit.java
package com.api.test_tcc_mobile.retrofit;

import com.api.test_tcc_mobile.model.User;

import okhttp3.ResponseBody;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.GET;
import retrofit2.http.POST;

public interface UserRetrofit {

    @POST("/login")
    Call<ResponseBody> authenticateUser(@Body User user);

    @GET("/get-current-user")
    Call<User> getCurrentUser();
}

File: kotlinTcc/app/src/main/java/com/api/kotlintcc/retrofit/UserRetrofit.kt
package com.api.kotlintcc.retrofit

import com.api.kotlintcc.model.User
import okhttp3.ResponseBody
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.GET
import retrofit2.http.POST

interface UserRetrofit {

    @POST("/login")
    fun authenticatorUser(@Body user: User): Call<ResponseBody>

    @GET("/get-current-user")
    fun getCurrentUser(): Call<User>
}